src/test/java/org/jenkins/plugins/lockableresources/LockStepTest.java

```diff
@@ -345,21 +347,23 @@ public void parallelLock() throws Exception {
                 + "	}\n"
                 + "}\n"));
 
-    WorkflowRun b1 = p.scheduleBuild2(0).waitForStart();
+    QueueTaskFuture<WorkflowRun> r1 = p.scheduleBuild2(0);
+    WorkflowRun b1 = r1.waitForStart();
     SemaphoreStep.waitForStart("wait-b/1", b1);
     // both messages are in the log because branch b acquired the lock and branch a is waiting to
     // lock
-    j.waitForMessage("[b] Lock acquired on [resource1]", b1);
+    j.waitForMessage("Lock acquired on [resource1]", b1);
     j.waitForMessage(
-        "[a] [resource1] is locked by " + b1.getFullDisplayName() + ", waiting...", b1);
+        "[resource1] is locked by " + b1.getFullDisplayName() + ", waiting...", b1);
     isPaused(b1, 2, 1);
 
     SemaphoreStep.success("wait-b/1", null);
 
-    j.waitForMessage("[a] Lock acquired on [resource1]", b1);
+    j.waitForMessage("Lock acquired on [resource1]", b1);
     isPaused(b1, 2, 0);
 
     assertNotNull(LockableResourcesManager.get().fromName("resource1"));
+    j.assertBuildStatusSuccess(r1);
```

Comment:
Would it help to wait some period of time for the Jenkins instance to become idle?  In this example, it would wait up to 9 seconds before ending the test.
```suggestion
    j.assertBuildStatusSuccess(r1);
    j.waitUntilNoActivityUpTo(9001); // Allow up to 9 seconds for job to finish
```